algorithmic_efficiency/workloads/criteo1tb/input_pipeline.py

```diff
@@ -9,9 +9,16 @@
 import os
 from typing import Optional, Sequence
 
-import jax
 import tensorflow as tf
 
+from algorithmic_efficiency import data_utils
+from algorithmic_efficiency.pytorch_utils import pytorch_setup
+
+RANK = pytorch_setup()[1]
+AUTOTUNE = tf.data.AUTOTUNE if RANK == 0 else None
```

Comment:
This condition is not necessary here since the input pipeline is only created in the process with `RANK == 0`.